src/main/org/tvrenamer/controller/util/FileUtilities.java

```diff
@@ -128,6 +128,16 @@ public static boolean areSameDisk(Path pathA, Path pathB) {
         }
     }
 
+    public static boolean isSameFile(final Path path1, final Path path2) {
```

Comment:
This method has been added but never used? Also just proxies to `Files.isSameFile()`. Shouldn't the `catch` be in `Files.isSameFile()` to ensure a `boolean` is always returned?